src/Microsoft.Health.Fhir.Core/Features/ArtifactStore/OciArtifactInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System;
+
+namespace Microsoft.Health.Fhir.Core.Features.ArtifactStore
+{
+    public class OciArtifactInfo
```

Comment:
>Oci [](http://example.com/codeflow?start=17&length=3)

Oracle cloud infrastructure?

This is what I'm getting if I search 'oci azure'

Reply:
Should read PR description.
Any chance you can add [https://docs.microsoft.com/en-us/azure/container-registry/container-registry-oci-artifacts](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-oci-artifacts) as comment in this file?

Reply:
Sure. I have added the OCI description to comments.